src/test/resources/alma-fix/990124590390206441.json

```diff
@@ -69,10 +69,6 @@
     "label" : "Deutsch"
   } ],
   "extent" : "1 CD-ROM ; 12 cm + 1 Beih. [6] S.",
-  "natureOfContent" : [ {
-    "label" : "CD-ROM",
-    "id" : "https://d-nb.info/gnd/4139307-7"
-  } ],
   "subject" : [ {
```

Comment:
How about moving this to `medium` instead of just removing it?

Reply:
I agree, but I would do this in context of: #1549